src/main/java/org/folio/circulation/resources/RequestScheduledNoticeProcessingResource.java

```diff
@@ -0,0 +1,42 @@
+package org.folio.circulation.resources;
+
+import java.util.Collection;
+import java.util.concurrent.CompletableFuture;
+
+import org.folio.circulation.domain.MultipleRecords;
+import org.folio.circulation.domain.notice.schedule.RequestScheduledNoticeHandler;
+import org.folio.circulation.domain.notice.schedule.ScheduledNotice;
+import org.folio.circulation.domain.notice.schedule.ScheduledNoticesRepository;
+import org.folio.circulation.support.Clients;
+import org.folio.circulation.support.Result;
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeZone;
+
+import io.vertx.core.http.HttpClient;
+
+public class RequestScheduledNoticeProcessingResource extends ScheduledNoticeProcessingResource {
+
+  private static final int SCHEDULED_NOTICES_PROCESSING_LIMIT = 100;
+
+  public RequestScheduledNoticeProcessingResource(HttpClient client) {
+    super("/circulation/request-scheduled-notices-processing" ,client);
+  }
+
+  @Override
+  protected CompletableFuture<Result<MultipleRecords<ScheduledNotice>>> findNoticesToSend(
+    ScheduledNoticesRepository scheduledNoticesRepository) {
+
+    final DateTime systemTime = DateTime.now(DateTimeZone.UTC);
+    return scheduledNoticesRepository.findRequestNoticesToSend(systemTime, SCHEDULED_NOTICES_PROCESSING_LIMIT);
+  }
+
+  @Override
+  protected CompletableFuture<Result<Collection<ScheduledNotice>>> handleNotices(
+    Clients clients, Collection<ScheduledNotice> scheduledNotices) {
+
+    final RequestScheduledNoticeHandler requestScheduledNoticeHandler =
+      RequestScheduledNoticeHandler.using(clients);
+
+    return requestScheduledNoticeHandler.handleNotices(scheduledNotices);
+  }
```

Comment:
Is it necessary to use local var here?
just save a couple of lines
```
 return RequestScheduledNoticeHandler.using(clients).handleNotices(scheduledNotices);
```

Reply:
done